src/pages/volunteer/index.tsx

```diff
@@ -93,7 +93,7 @@ export default () => {
         paragraph3={{
           title: "Research & Community Development",
           description:
-            "To better understand the specific experiences people have endured with housing insecurity, our research team works alongside Community Representatives to better understand how best to serve these challenges.",
+            "No two experiences of people facing housing insecurity is the same. We have a dedicated team working alongside individuals who have experienced homelessness learning how best to serve the diverse challenges faced by those facing housing insecurity.",
```

Comment:
It looks like paragraphs 2 and 3 are swapped compared to the new page designs, since Research & Community Development should now be the 2nd paragraph.